src/mkdocstrings_handlers/python/templates/material/language.html

```diff
@@ -1,6 +1,12 @@
 <!-- Import translations for given language and fallback -->
 
-{% import "languages/" ~ locale ~ ".html" as lang %}
+{% set lang_pth = "languages/" ~ locale ~ ".html" %}
+{% if lang_pth is existing_template %}
+    {% import lang_pth as lang %}
```

Comment:
What happens when we try to import without checking existence first?

Reply:
TemplateNotFound exception, if user sets an unsupported locale

Reply:
Thanks. I've opened this: https://github.com/pallets/jinja/issues/1862, let see what they say :slightly_smiling_face:

Reply:
Even if the feature is implemented in Jinja, I don't think we'll be able to use it anytime soon, so lets move forward :slightly_smiling_face: 

I'll run a last review and commit some fixes directly to the PR if you don't mind, this coming week.

Reply:
Would appreciate it, thanks!